seed/challenges/01-responsive-web-design/basic-css.json

```diff
@@ -1290,9 +1290,11 @@
       "id": "bad87fee1348bd9aedf08815",
       "title": "Make Circular Images with a border-radius",
       "description": [
-        "In addition to pixels, you can also specify a <code>border-radius</code> using a percentage.",
+        "In addition to pixels, you can also specify the <code>border-radius</code> using a percentage.",
         "<hr>",
-        "Give your cat photo a <code>border-radius</code> of <code>50%</code>."
+        "Give your cat photo a <code>border-radius</code> of <code>50%</code>.",
+        "<hr>",
+        "As you can tell from the end result, this is an easy way to create circular images."
```

Comment:
Opinionated verbiage is not recommend.

Reply:
I agree. We shouldn't be presumptive and say things like "as you can tell" because - what if the user can't tell? They'll feel dumb. We can probably remove line 1297 completely imho.

Reply:
Agreed - this sentence is presumptive (what if the user can't tell that) and also this sentence probably isn't necessary imho.